pkg/test/expectations/expectations.go

```diff
@@ -358,6 +358,16 @@ func ExpectProvisionedNoBinding(ctx context.Context, c client.Client, cluster *s
 	return bindings
 }
 
+func ExpectProvisionedResults(ctx context.Context, c client.Client, cluster *state.Cluster, cloudProvider cloudprovider.CloudProvider, provisioner *provisioning.Provisioner, pods ...*corev1.Pod) scheduling.Results {
```

Comment:
This is so specific to your use-case -- does it make sense to have this as an expectation at all or should we just put this directly in the test?